docs/space-amp.md

```diff
@@ -0,0 +1,57 @@
+## Space Amp
```

Comment:
This needs to be an `h1`, which changes all subsequent levels

Reply:
Odd.  Dunno why I had that as h2.